packages/experimental-utils/src/ts-eslint/Rule.ts

```diff
@@ -1,5 +1,5 @@
 import { ParserServices, TSESTree } from '@typescript-eslint/typescript-estree';
-import { JSONSchema4 } from 'json-schema';
+import { JSONSchema4 } from 'json-schema'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Is this lint comment needed now that it is in dependencies? (I have never used this plugin or rule before so genuinely asking)

Reply:
@JamesHenry Try running ESLint with `--report-unused-disable-directives` flag-- if the disable comment isn't doing anything, you'll get an extra lint message.

Reply:
@platinumazure - `eslint-plugin-eslint-comments` provides a rule `no-unused-disable` which checks this at lint time without any flags.

Reply:
this one in particular is because the module is actually called `@types/json-schema`, which doesn't match up with the import (there isn't any actual `json-schema` module to install).

I've created an export in the package to fix this.